challenge.rb

```diff
@@ -20,7 +20,7 @@ def fetch_the_first_two(input)
 end
 
 def fetch_CD_animals(input)
-  input.select{|i| i.start_with?("c") || i.start_with?("d")}
+  input.grep(/\Ac|\Ad/)
```

Comment:
Nice.  `grep(/\A[cd]/)` might be a bit cleaner, but you've got the point, good usage of `\A`